core/src/main/java/com/scalar/db/api/Delete.java

```diff
@@ -122,6 +124,12 @@ public Delete withCondition(MutationCondition condition) {
     return (Delete) super.withCondition(condition);
   }
 
+  @Nonnull
+  @Override
+  public Optional<MutationCondition> getCondition() {
```

Comment:
[trivial] This can be removed?

Reply:
Actually, `Mutation.getCondition()` is deprecated and will be removed in the future. However, `Delete.getCondition()` is not deprecated and will remain. Therefore, we don't add `@Deprecated` to this method.